Can't loginafter changing language to Turkish

### Bugs

I changed my email from the team panel, and after changing my language as well, I can't login to Metabase. I get the error `Assert failed: (integer? (:id user))`.

- Your browser and the version: Chrome 70.0.3538.102 (Official Build) (64-bit)
- Your operating system: OS X 10.14.1
- Your databases: MongoDB
- Metabase version: 0.31.1
- Metabase hosting environment: Mac app
- Metabase internal database: H2

Steps to reproduce the bug:
- Log in to Metabase.
- Change your email address from admin panel.
- Change your language (in case it might be related to the language; change it to Turkish). You'll be automatically signed off.
- Try to log in again, you should get an error like this:

![image](https://user-images.githubusercontent.com/13895224/48944212-dd9b1200-ef36-11e8-8a55-804bc3dac45f.png)

I've reproduced this. It only happens with Turkish, which is odd. Looks like it has something to do with this string:

```
12-05 00:42:18 ERROR util.i18n :: Unable to translate string 'User lookup filter, the placeholder '{login}' will be replaced by the user supplied login.'
java.lang.IllegalArgumentException: can't parse argument number: login
	at java.text.MessageFormat.makeFormat(MessageFormat.java:1429)
	at java.text.MessageFormat.applyPattern(MessageFormat.java:479)
	at java.text.MessageFormat.<init>(MessageFormat.java:380)
	at puppetlabs.i18n.core$fmt.invoke(core.clj:250)
	at puppetlabs.i18n.core$translate.doInvoke(core.clj:257)
	at clojure.lang.RestFn.invoke(RestFn.java:445)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:663)
	at clojure.core$apply.invoke(core.clj:652)
	at metabase.util.i18n$translate.invokeStatic(i18n.clj:27)
	at metabase.util.i18n$translate.invoke(i18n.clj:20)
	at metabase.util.i18n.UserLocalizedString.toString(i18n.clj:37)
	at clojure.core$str.invokeStatic(core.clj:553)
	at clojure.core$str.invoke(core.clj:544)
	at metabase.models.setting$user_facing_info.invokeStatic(setting.clj:620)
	at metabase.models.setting$user_facing_info.invoke(setting.clj:610)
	at metabase.models.setting$all$iter__17084__17088$fn__17089.invoke(setting.clj:630)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.Cons.next(Cons.java:39)
	at clojure.lang.RT.next(RT.java:706)
	at clojure.core$next__5108.invokeStatic(core.clj:64)
	at clojure.core$next__5108.invoke(core.clj:64)
	at cheshire.generate$generate.invokeStatic(generate.clj:132)
	at cheshire.generate$generate.invoke(generate.clj:116)
	at cheshire.core$generate_stream.invokeStatic(core.clj:105)
	at cheshire.core$generate_stream.invoke(core.clj:80)
	at metabase.core$streamed_json_response$fn__62582.invoke(core.clj:55)
	at ring.util.io$piped_input_stream$fn__14844.invoke(io.clj:28)
	at clojure.core$binding_conveyor_fn$fn__5476.invoke(core.clj:2022)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NumberFormatException: For input string: "login"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at java.text.MessageFormat.makeFormat(MessageFormat.java:1427)
	... 35 more
```

--- a/project.clj
+++ b/project.clj
@@ -122,7 +122,7 @@
    [org.eclipse.jetty/jetty-server "9.4.15.v20190215"]                ; We require JDK 8 which allows us to run Jetty 9.4, ring-jetty-adapter runs on 1.7 which forces an older version
    [ring/ring-json "0.4.0"]                                           ; Ring middleware for reading/writing JSON automatically
    [stencil "0.5.0"]                                                  ; Mustache templates for Clojure
-   [toucan "1.12.0" :exclusions [org.clojure/java.jdbc honeysql]]     ; Model layer, hydration, and DB utilities
+   [toucan "1.13.0" :exclusions [org.clojure/java.jdbc honeysql]]     ; Model layer, hydration, and DB utilities
    [weavejester/dependency "0.2.1"]]                                  ; Dependency graphs and topological sorting
 
   :main ^:skip-aot metabase.core

